src/core/device.c

```diff
@@ -709,16 +709,34 @@ static void device_update_found_by_name(Manager *m, const char *path, DeviceFoun
 }
 
 static bool device_is_ready(sd_device *dev) {
+        int r;
+
         assert(dev);
 
-        if (device_is_renaming(dev) > 0)
+        r = device_is_renaming(dev);
+        if (r < 0)
+                log_device_error_errno(dev, r, "Failed to check if device is renaming: %m");
+        if (r > 0) {
+                log_device_debug(dev, "Device busy: renaming");
```

Comment:
I guess this is quite noisy. Please drop it or introduce `log_device_trace()` and use it.

Reply:
At least in a mkosi QEMU VM it never fires, but maybe that's not representative of all scenarios

Reply:
I guess this is (or will be) repeatedly called for the same device. But let's see how often it is logged.